app/views/idv/sessions/index.html.slim

```diff
@@ -24,7 +24,7 @@
       class: 'block col-12 field', input_html: { 'aria-labelledby': 'label-dob' }
   h2#label-ssn.heading = t('idv.titles.session.ssn')
   .mb4.sm-col-5
-    = f.input :ssn, label: false, required: true, pattern: '[0-9]*',
+    = f.input :ssn, label: false, required: true, pattern: '^\d{3}-?\d{2}-?\d{4}$',
```

Comment:
Is it worth defining these patterns in code somewhere?  I'd like to be able to add comments/documentation for future readers so they understand why we chose certain values.